app/recordtransfer/templates/recordtransfer/signupform.html

```diff
@@ -41,9 +35,10 @@
                 </div>
             {% endfor %}
             <div class="flex-item">
-                <label></label>
+                {{ field.label_tag }}
```

Comment:
Due to an unfortunate styling decision I made in the past, this is actually required to be an empty label:

```suggestion
                <label></label>
```

Additionally, `{{ field }}` is something that exists only in the for loop, there's no `{{ field }}` that's defined here.

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/561/commits/fb2aa83c8e3a11340cbdc5b35878dbd226b64431